packages/contracts/contracts/TellerV2.sol

```diff
@@ -488,6 +499,16 @@ contract TellerV2 is
         // Declare the bid acceptor as the lender of the bid
         bid.lender = sender;
 
+        // Set payment cycle value to current day if market setting is based on monthly payments
+        if (
+            bidPaymentCycleType[_bidId] ==
```

Comment:
what happens if a loan is opened up on the 29th of february?  etc etc

Reply:
added a conditional to adjust the date if it's a leap year and the lastRepaidTimestamp is on the 29th of feb